client/webserver/locales/pl-pl.go

```diff
@@ -177,6 +177,7 @@ var PlPL = map[string]string{
 	"how_reg":                     "Jak chcesz uiścić opłatę rejestracyjną?",
 	"All markets at":              "Wszystkie rynki na",
 	"pick a different asset":      "wyierz inne aktywo",
+	"Create":                      "Utwórz",
```

Comment:
@artikozel Is this one ok?

Reply:
Yep! Either "Stwórz" or "Utwórz" are fine.